tests/json/schema-1-tamps/input.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import psi4
+import json
+
+#Generate JSON data
+method = "CCSD"
+basis = "sto-3g"
+json_data = {
+    "schema_name": "qcschema_input",
+    "schema_version": 1,
+    "return_output": True,
+    "molecule": {
+        "geometry": [
+            0.0,
+            0.0,
+            -0.1294769411935893,
+            0.0,
+            -1.494187339479985,
+            1.0274465079245698,
+            0.0,
+            1.494187339479985,
+            1.0274465079245698
+        ],
+        "symbols": [
+          "O",
+          "H",
+          "H"
+        ],
+        "connectivity" : [
+            (0, 1, 1.0),
+            (0, 2, 1.0)
+        ],
+        "fix_symmetry": "c1"
+    },
+    "driver": "energy",
+    "model": {
+        "method": method,
+        "basis": basis
+    },
+    "keywords": {
+        "scf_type": "df",
+        "mp2_type": "df"
+    },
+    "extras": {
+        "psi4:arrays": {
+            "tIjAb": [],
+            "tIA": [],
+            "Da": []
+        }
+    }
+}
+json_ret = psi4.json_wrapper.run_json_qcschema(json_data, True, True)
+
+h2o_test = psi4.geometry("""
+0 1
+O 0.0 0.0 0.1294769411935893
+H 0.0 -1.494187339479985 1.0274465079245698
+H 0.0 1.494187339479985 1.0274465079245698
+symmetry c1
+""")
+e, wfn = psi4.energy(f"{method}/{basis}", return_wfn = True)
```

Comment:
I don't know what's going on here. For some reason, the amplitudes are not being set properly in this call to the energy. The dimensions are right, but all the values are zero. The data from the schema is being set properly and upon inspection, seems to match what is given in the output files. But direct calls are failing.

Reply:
fyi, there's a report that amplitutes are/aren't available depending on if basis is set by option or set in the energy call in mtd/basis format. you might try the "set basis" route to check if that's what you're seeing.

Reply:
And if that is the problem, please file an issue about the amplitude availability. I haven't heard of this problem before.

Reply:
that amplitude availability issue I think had to do with the mtd/basis routing through cbs to process the method string, and cbs() has clean statements btwn each part, whereas energy() doesn't. I don't remember if it's solved by ddd, but it should at least be uniform there.